_docs_integrate/attribute-introduction.md

```diff
@@ -84,51 +89,55 @@ The [RelationshipAttribute value types]({% link _docs_integrate/attribute-values
-
-In the context of sharing a RelationshipAttribute with peers who are not involved in the Relationship in which the RelationshipAttribute exists, an own shared RelationshipAttribute that is originated from another own shared RelationshipAttribute is also referred to as an **own shared ThirdPartyRelationshipAttribute**. Its counterpart is then also referred to as a **peer shared ThirdPartyRelationshipAttribute**. Since third party owned RelationshipAttributes always originate from peer shared RelationshipAttributes, it is not at all necessary to introduce the term third party owned ThirdPartyRelationshipAttributes.
-{: .notice--info}
-
-<div style="width: 640px; height: 720px; margin: 10px; position: relative;"><iframe allowfullscreen frameborder="0" style="width:640px; height:720px" src="https://lucid.app/documents/embedded/dd79b31c-0404-48bb-9773-9b989801c03c" id="RbDyLSjh7BzY"></iframe></div>
+From a technical perspective, a RelationshipAttribute is always stored as the `content` of a [LocalAttribute]({% link _docs_integrate/data-model-overview.md %}#localattribute). Depending on the values of certain properties of the LocalAttribute, a LocalAttribute whose `content` is given by a RelationshipAttribute is also referred to as an **own shared RelationshipAttribute**, a **peer shared RelationshipAttribute**, an **own shared ThirdPartyRelationshipAttribute**, a **peer shared ThirdPartyRelationshipAttribute** or a **third party owned RelationshipAttribute**. For the simple initial creation of a RelationshipAttribute within a given Relationship, the terms [own shared RelationshipAttribute and peer shared RelationshipAttribute]({% link _docs_integrate/attribute-introduction.md %}#own-shared-and-peer-shared-relationshipattributes) are relevant. The other terms for LocalAttributes in connection with RelationshipAttributes are needed if an existing RelationshipAttribute from one Relationship is shared with a peer from another Relationship.
 
-#### Own shared RelationshipAttributes
+#### Own shared and peer shared RelationshipAttributes
 
 A [RelationshipAttribute]({% link _docs_integrate/data-model-overview.md %}#relationshipattribute) can only exist in the context of a Relationship and must therefore be stored locally for both Identities involved in the Relationship.
 Accordingly, the [creation of a RelationshipAttribute]({% link _docs_integrate/create-attributes-for-yourself.md %}#create-a-relationshipattribute) corresponds to the creation of one [LocalAttribute]({% link _docs_integrate/data-model-overview.md %}#localattribute) for its `owner` and one LocalAttribute for the peer with whom the `owner` has established a Relationship.
```

Comment:
I have only found this formulation once more, in the Data Model Overview, and adapted it there. Have you noticed any other places in which this may need to be adapted?

Reply:
Line 97, 102, 128 of this file.